.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -0,0 +1,71 @@
+name: Azure Functions VSCode Web Bug Report
+description: File a Azure Functions VSCode bug report
+title: "Bug Title Here"
+body:
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+      placeholder: What should have occurred?
+
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual Behavior
+      description: A clear and concise description of what actually happened.
+      placeholder: What went wrong?
+
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to Reproduce
+      description: Please provide detailed step-by-step instructions on how to reproduce the bug.
+      placeholder: |
+        1. Go to the [specific page or section] in the application.
+        2. Click on [specific button or link].
+        3. Scroll down to [specific location].
+        4. Observe [describe what you see, e.g., an error message or unexpected behavior].
+        5. Include any additional steps or details that may be relevant.
+
+  - type: input
+    id: function-app-name
+    attributes:
+      label: Function App Name
+      description: Please give the name of the function app being used.
+
+  - type: input
+    id: tunnel-id
+    attributes:
+      label: Tunnel ID
+      description: If applicable, please provide the Tunnel ID. It is located in the error message or at the edge of the status bar, typically in the form [word]-[word]-[id].[region]. Example - joyful-horse-y7jqra5.use
+
```

Comment:
Do you think asking region would help? or tunnel id should be good for now.

Reply:
The ID will include the region suffix as a part of it. Also the function app name will give this away too